src/index.ts

```diff
@@ -5,13 +5,15 @@ export * from './authorization_service_configuration';
 export * from './crypto_utils';
 export * from './errors';
 export * from './flags';
+export * from './jquery_requestor';
 export * from './logger';
 export * from './query_string_utils';
 export * from './redirect_based_handler';
+export * from './requestor';
 export * from './revoke_token_request';
 export * from './storage';
+export * from './test_requestor';
```

Comment:
No need to export the `test_requestor`. I think it was only used in the context of unit tests.